scripts/taiseilib/version.py

```diff
@@ -1,6 +1,5 @@
-
+#!/usr/bin/env python3
```

Comment:
You should just revert this file with `git checkout master -- scripts/taiseilib/version.py`
The shebang and +x permissions aren't needed.